apps/site/lib/site_web/templates/cms/paragraph/_column_multi.html.eex

```diff
@@ -9,16 +9,19 @@
   </div>
 <% end %>
 
-<%= extend_width_if has_cards?, :cards do %>
-  <div class="c-multi-column__row row">
-    <%= for column <- @content.columns do %>
-      <div class="c-multi-column__column col-sm-6 col-md-<%= grid(@content) %>">
 
-        <%= for paragraph <- column.paragraphs do %>
-          <%= render_paragraph_content(paragraph, @conn) %>
-        <% end %>
+<%= if length(@content.columns) > 0 do %>
```

Comment:
It's minor, but `Enum.any?(things)` is more efficient than `length(things) > 0`, which has to iterate the whole list to find the length.